crates/bevy_scene/src/command.rs

```diff
@@ -14,7 +14,9 @@ pub struct SpawnScene {
 
 impl Command for SpawnScene {
     fn write(self, world: &mut World) {
-        let mut spawner = world.get_resource_mut::<SceneSpawner>().unwrap();
+        let mut spawner = world
+            .get_resource_mut::<SceneSpawner>()
+            .expect("Could not get `SceneSpawner` resource in the `World`");
```

Comment:
```suggestion
            .expect("Could not find `SceneSpawner` resource in the `World`. Did you forget to add `ScenePlugin` to your `App`?");
```

Reply:
And elsewhere.

Reply:
Okay, even for something like: `TypeRegistryArc`, is it supposed to be added by the user?